src/toil/server/app.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2015-2021 Regents of the University of California
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import logging
+import os
+
+import connexion  # type: ignore
+
+from toil.server.wes.toil_backend import ToilBackend
+from toil.server.wsgi_app import run_app
+from toil.version import version
+
+logger = logging.getLogger(__name__)
+
+
+def parser_with_server_options() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(description="Toil server mode.")
+
+    parser.add_argument("--debug", action="store_true", default=False)
+    parser.add_argument("--host", type=str, default="0.0.0.0",
+                        help="The host interface that the Toil server binds on. (default: '0.0.0.0').")
```

Comment:
Shouldn't this default to `127.0.0.1` and/or `::1`? Otherwise by default we're giving everyone on the network access to your PC when you run this.